README.md

```diff
@@ -11,7 +11,7 @@ Summary: A Python module for hyperspectral image processing.
 
 A pure Python module for processing hyperspectral image data (imaging spectroscopy data).
 It has functions for reading, displaying, manipulating, and classifying hyperspectral
-imagery. SPy is Free, Open Source Software (FOSS) distributed under the MIT License.
+imagery. SPy is Free, Open Source Software (FOSS) distributed under the GNU General Public License.
```

Comment:
@tboggs 

When this PR was merged into master, it overwrite the license.

Reply:
I'm confused by the feedstock build process. Where did the GPL text come from? Should I not have accepted the bot's PR?

Reply:
> Where did the GPL text come from? 

I can only assume it was an automated mechanism which picked up the older license data from some outdated resource...

> Should I not have accepted the bot's PR?

All we need to do is update that text. I can do it right now :)

Reply:
@lewismc sorry to keep bugging you about this. Was the text updated? The description still references the GPL.

Reply:
@lewismc Could the fix be to update text in the [meta.yaml](https://github.com/conda-forge/spectral-feedstock/blob/master/recipe/meta.yaml#L41) file?